c2corg_ui/views/document.py

```diff
@@ -366,9 +366,9 @@ def render_page(content1, content2):
                 'version1': version1,
                 'version2': version2,
                 'geometry1': doc_v1['geometry']['geom']
-                if doc_v1['geometry'] else None,
+                if 'geometry' in doc_v1 else None,
```

Comment:
Are you sure your API is up?

I think I got this error when trying to load a document while my API was down. Or perhaps your API does not return a JSON response. Try to load in a browser the API request your UI is doing.